sdk/textanalytics/tests.yml

```diff
@@ -14,9 +14,6 @@ extends:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         Location: '${{ parameters.Location }}'
-      Canary:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-        Location: 'centraluseuap'
```

Comment:
Was this intentional? We provide needed backstop testing, so testing early against their early deployments is vital.

Reply:
While it was listed here, there were no tests actually being run against Canary. I've created an issue to follow up on this: https://github.com/Azure/azure-sdk-for-net/issues/35639